Source/AliveLib/MainMenu.hpp

```diff
@@ -282,7 +283,7 @@ class MainMenuController : public BaseAnimatedWithPhysicsGameObject
     __int16 field_228_res_idx;
     __int16 field_22A_anim_frame_num;
     int field_22C_T80;
-    __int16 field_230_fmv_level_index;
+    __int16 field_230_selected_demo_index;
```

Comment:
This is used for FMV and Demos at least from what I can see, I'd rename to demo_or_fmv or just selected_index or something like that